frontend/src/components/BestExpertiseSection/Card/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Button } from '../../shared/button/index';
+
+const Card = ({ title, details, button }) => {
+  return (
+    <article>
```

Comment:
why you used the `article` tag, not div?

Reply:
because its a stand-alone part of the section that have meaning in it self apart from the section.